unit_test/blas/Test_Blas2_team_gemv.hpp

```diff
@@ -1,3 +1,9 @@
+// Note: Luc Berger-Vergiat 04/14/21
+//       This tests uses KOKKOS_LAMBDA so we need
+//       to make sure that these are enabled in
+//       the CUDA backend before including this test.
+#if !defined(TEST_CUDA_BLAS_CPP) || defined(KOKKOS_ENABLE_CUDA_LAMBDA)
```

Comment:
I'm restoring `CUDA_LAMBDA` guards in `TeamGemv` and `TeamVectorGemv` tests: I thought lambdas appear only in tests (and will be gone after refactoring), but they also appear in the implementations. Added TODO in the description to fix that - on this or followup PR.